charts/soroban-rpc/values-pubnet.yaml

```diff
@@ -48,138 +48,159 @@ sorobanRpc:
-      - address: "core-live-c.stellar.org"
-        history: "curl -sf http://history.stellar.org/prd/core-live/core_live_003/{0} -o {1}"
-        homeDomain: "www.stellar.org"
-        name: "sdf_live_3"
-        publicKey: "GABMKJM6I25XI4K7U6XWMULOUQIQ27BCTMLS6BYYSOWKTBUXVRJSXHYQ"
-      - address: "stellar-full-validator1.bdnodes.net"
-        history: "curl -sf https://stellar-full-history1.bdnodes.net/{0} -o {1}"
-        homeDomain: "stellar.blockdaemon.com"
-        name: "blockdaemon_1"
-        publicKey: "GAAV2GCVFLNN522ORUYFV33E76VPC22E72S75AQ6MBR5V45Z5DWVPWEU"
-      - address: "stellar-full-validator2.bdnodes.net"
-        history: "curl -sf https://stellar-full-history2.bdnodes.net/{0} -o {1}"
-        homeDomain: "stellar.blockdaemon.com"
-        name: "blockdaemon_2"
-        publicKey: "GAVXB7SBJRYHSG6KSQHY74N7JAFRL4PFVZCNWW2ARI6ZEKNBJSMSKW7C"
-      - address: "stellar-full-validator3.bdnodes.net"
-        history: "curl -sf https://stellar-full-history3.bdnodes.net/{0} -o {1}"
-        homeDomain: "stellar.blockdaemon.com"
-        name: "Blockdaemon Validator 3"
-        publicKey: "GAYXZ4PZ7P6QOX7EBHPIZXNWY4KCOBYWJCA4WKWRKC7XIUS3UJPT6EZ4"
-      - address: "finland.stellar.coinqvest.com"
-        history: "curl -sf https://finland.stellar.coinqvest.com/history/{0} -o {1}"
-        homeDomain: "coinqvest.com"
-        name: "coinqvest_fi"
-        publicKey: "GADLA6BJK6VK33EM2IDQM37L5KGVCY5MSHSHVJA4SCNGNUIEOTCR6J5T"
-      - address: "hongkong.stellar.coinqvest.com"
-        history: "curl -sf https://hongkong.stellar.coinqvest.com/history/{0} -o {1}"
-        homeDomain: "coinqvest.com"
-        name: "coinqvest_hk"
-        publicKey: "GAZ437J46SCFPZEDLVGDMKZPLFO77XJ4QVAURSJVRZK2T5S7XUFHXI2Z"
-      - address: "germany.stellar.coinqvest.com"
-        history: "curl -sf https://germany.stellar.coinqvest.com/history/{0} -o {1}"
-        homeDomain: "coinqvest.com"
-        name: "coinqvest_de"
-        publicKey: "GD6SZQV3WEJUH352NTVLKEV2JM2RH266VPEM7EH5QLLI7ZZAALMLNUVN"
-      - address: "stellar-de-fra.satoshipay.io"
-        history: "curl -sf https://stellar-history-de-fra.satoshipay.io/{0} -o {1}"
-        homeDomain: "satoshipay.io"
-        name: "satoshipay_de"
-        publicKey: "GC5SXLNAM3C4NMGK2PXK4R34B5GNZ47FYQ24ZIBFDFOCU6D4KBN4POAE"
-      - address: "stellar-sg-sin.satoshipay.io"
-        history: "curl -sf https://stellar-history-sg-sin.satoshipay.io/{0} -o {1}"
-        homeDomain: "satoshipay.io"
-        name: "satoshipay_sg"
-        publicKey: "GBJQUIXUO4XSNPAUT6ODLZUJRV2NPXYASKUBY4G5MYP3M47PCVI55MNT"
-      - address: "stellar-us-iowa.satoshipay.io"
-        history: "curl -sf https://stellar-history-us-iowa.satoshipay.io/{0} -o {1}"
-        homeDomain: "satoshipay.io"
-        name: "satoshipay_us"
-        publicKey: "GAK6Z5UVGUVSEK6PEOCAYJISTT5EJBB34PN3NOLEQG2SUKXRVV2F6HZY"
-      - address: "bootes.publicnode.org"
+    validators
```

Comment:
yaml typo, s/b `validators:`

retested helm with that change on local copy, output looks good:
```
$ helm template soroban-test ./charts/soroban-rpc --values /Users/sreuland/Downloads/values-pubnet.yaml 
---
# Source: soroban-rpc/templates/soroban-rpc-cm.yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: soroban-test-soroban-rpc
  namespace: default
  labels:
    app: soroban-test-soroban-rpc
    chart: soroban-rpc-0.1.1
    release: soroban-test
    heritage: Helm
data:
  soroban-rpc.cfg: |
    CAPTIVE_CORE_CONFIG_PATH="/config/stellar-captive-core.cfg"
    CAPTIVE_CORE_STORAGE_PATH="/var/lib/stellar/captive-core"
    CAPTIVE_CORE_USE_DB=true
    STELLAR_CORE_BINARY_PATH="/usr/bin/stellar-core"
    ENDPOINT="0.0.0.0:8000"
    ADMIN_ENDPOINT="0.0.0.0:6061"
    DB_PATH="/var/lib/stellar/soroban-rpc-db.sqlite"
    STELLAR_CAPTIVE_CORE_HTTP_PORT=11626
    FRIENDBOT_URL="https://friendbot.stellar.org/"
    NETWORK_PASSPHRASE="Public Global Stellar Network ; September 2015"
    HISTORY_ARCHIVE_URLS="https://stellar-history-usw.franklintempleton.com/azuswshf401,https://stellar-archive-1-lobstr.s3.amazonaws.com,https://hercules-history.publicnode.org,https://stellar-full-history1.bdnodes.net,https://history.stellar.org/prd/core-live/core_live_003,https://history.stellar.org/prd/core-live/core_live_002,https://history.stellar.org/prd/core-live/core_live_001"
    CHECKPOINT_FREQUENCY=64
  stellar-captive-core.cfg: |
    DATABASE="sqlite3:///var/lib/stellar/stellar-captive-core.db"
    HTTP_PORT=11626
    PUBLIC_HTTP_PORT=true
    PEER_PORT=11625
    EXPERIMENTAL_BUCKETLIST_DB=true
    EXPERIMENTAL_BUCKETLIST_DB_INDEX_PAGE_SIZE_EXPONENT=12
    ENABLE_SOROBAN_DIAGNOSTIC_EVENTS=true
    NETWORK_PASSPHRASE="Public Global Stellar Network ; September 2015"
    UNSAFE_QUORUM=true
    
    [[HOME_DOMAINS]]
    HOME_DOMAIN="publicnode.org"
    QUALITY="HIGH"
    [[HOME_DOMAINS]]
    HOME_DOMAIN="lobstr.co"
    QUALITY="HIGH"
    [[HOME_DOMAINS]]
    HOME_DOMAIN="www.franklintempleton.com"
    QUALITY="HIGH"
    [[HOME_DOMAINS]]
    HOME_DOMAIN="satoshipay.io"
    QUALITY="HIGH"
    [[HOME_DOMAINS]]
    HOME_DOMAIN="whalestack.com"
    QUALITY="HIGH"
    [[HOME_DOMAINS]]
    HOME_DOMAIN="www.stellar.org"
    QUALITY="HIGH"
    [[HOME_DOMAINS]]
    HOME_DOMAIN="stellar.blockdaemon.com"
    QUALITY="HIGH"
    [[VALIDATORS]]
    NAME="Boötes"
    HOME_DOMAIN="publicnode.org"
    PUBLIC_KEY=""
    ADDRESS="bootes.publicnode.org:11625"
    HISTORY="curl -sf https://bootes-history.publicnode.org/{0} -o {1}"
    [[VALIDATORS]]
    NAME="Lyra by BP Ventures"
    HOME_DOMAIN="publicnode.org"
    PUBLIC_KEY=""
    ADDRESS="lyra.publicnode.org:11625"
    HISTORY="curl -sf https://lyra-history.publicnode.org/{0} -o {1}"
    [[VALIDATORS]]
    NAME="Hercules by OG Technologies"
    HOME_DOMAIN="publicnode.org"
    PUBLIC_KEY=""
    ADDRESS="hercules.publicnode.org:11625"
    HISTORY="curl -sf https://hercules-history.publicnode.org/{0} -o {1}"
    [[VALIDATORS]]
    NAME="LOBSTR 3 (North America)"
    HOME_DOMAIN="lobstr.co"
    PUBLIC_KEY=""
    ADDRESS="v3.stellar.lobstr.co:11625"
    HISTORY="curl -sf https://archive.v3.stellar.lobstr.co/{0} -o {1}"
    [[VALIDATORS]]
    NAME="LOBSTR 1 (Europe)"
    HOME_DOMAIN="lobstr.co"
    PUBLIC_KEY=""
    ADDRESS="v1.stellar.lobstr.co:11625"
    HISTORY="curl -sf https://archive.v1.stellar.lobstr.co/{0} -o {1}"
    [[VALIDATORS]]
    NAME="LOBSTR 2 (Europe)"
    HOME_DOMAIN="lobstr.co"
    PUBLIC_KEY=""
    ADDRESS="v2.stellar.lobstr.co:11625"
    HISTORY="curl -sf https://archive.v2.stellar.lobstr.co/{0} -o {1}"
    [[VALIDATORS]]
    NAME="LOBSTR 4 (Asia)"
    HOME_DOMAIN="lobstr.co"
    PUBLIC_KEY=""
    ADDRESS="v4.stellar.lobstr.co:11625"
    HISTORY="curl -sf https://archive.v4.stellar.lobstr.co/{0} -o {1}"
    [[VALIDATORS]]
    NAME="LOBSTR 5 (India)"
    HOME_DOMAIN="lobstr.co"
    PUBLIC_KEY=""
    ADDRESS="v5.stellar.lobstr.co:11625"
    HISTORY="curl -sf https://archive.v5.stellar.lobstr.co/{0} -o {1}"
    [[VALIDATORS]]
    NAME="FT SCV 2"
    HOME_DOMAIN="www.franklintempleton.com"
    PUBLIC_KEY=""
    ADDRESS="stellar2.franklintempleton.com:11625"
    HISTORY="curl -sf https://stellar-history-usc.franklintempleton.com/azuscshf401/{0} -o {1}"
    [[VALIDATORS]]
    NAME="FT SCV 3"
    HOME_DOMAIN="www.franklintempleton.com"
    PUBLIC_KEY=""
    ADDRESS="stellar3.franklintempleton.com:11625"
    HISTORY="curl -sf https://stellar-history-ins.franklintempleton.com/azinsshf401/{0} -o {1}"
    [[VALIDATORS]]
    NAME="FT SCV 1"
    HOME_DOMAIN="www.franklintempleton.com"
    PUBLIC_KEY=""
    ADDRESS="stellar1.franklintempleton.com:11625"
    HISTORY="curl -sf https://stellar-history-usw.franklintempleton.com/azuswshf401/{0} -o {1}"
    [[VALIDATORS]]
    NAME="SatoshiPay Frankfurt"
    HOME_DOMAIN="satoshipay.io"
    PUBLIC_KEY=""
    ADDRESS="stellar-de-fra.satoshipay.io:11625"
    HISTORY="curl -sf https://stellar-history-de-fra.satoshipay.io/{0} -o {1}"
    [[VALIDATORS]]
    NAME="SatoshiPay Singapore"
    HOME_DOMAIN="satoshipay.io"
    PUBLIC_KEY=""
    ADDRESS="stellar-sg-sin.satoshipay.io:11625"
    HISTORY="curl -sf https://stellar-history-sg-sin.satoshipay.io/{0} -o {1}"
    [[VALIDATORS]]
    NAME="SatoshiPay Iowa"
    HOME_DOMAIN="satoshipay.io"
    PUBLIC_KEY=""
    ADDRESS="stellar-us-iowa.satoshipay.io:11625"
    HISTORY="curl -sf https://stellar-history-us-iowa.satoshipay.io/{0} -o {1}"
    [[VALIDATORS]]
    NAME="Whalestack (Germany)"
    HOME_DOMAIN="whalestack.com"
    PUBLIC_KEY=""
    ADDRESS="germany.stellar.whalestack.com:11625"
    HISTORY="curl -sf https://germany.stellar.whalestack.com/history/{0} -o {1}"
    [[VALIDATORS]]
    NAME="Whalestack (Hong Kong)"
    HOME_DOMAIN="whalestack.com"
    PUBLIC_KEY=""
    ADDRESS="hongkong.stellar.whalestack.com:11625"
    HISTORY="curl -sf https://hongkong.stellar.whalestack.com/history/{0} -o {1}"
    [[VALIDATORS]]
    NAME="Whalestack (Finland)"
    HOME_DOMAIN="whalestack.com"
    PUBLIC_KEY=""
    ADDRESS="finland.stellar.whalestack.com:11625"
    HISTORY="curl -sf https://finland.stellar.whalestack.com/history/{0} -o {1}"
    [[VALIDATORS]]
    NAME="SDF 2"
    HOME_DOMAIN="www.stellar.org"
    PUBLIC_KEY=""
    ADDRESS="core-live-b.stellar.org:11625"
    HISTORY="curl -sf http://history.stellar.org/prd/core-live/core_live_002/{0} -o {1}"
    [[VALIDATORS]]
    NAME="SDF 1"
    HOME_DOMAIN="www.stellar.org"
    PUBLIC_KEY=""
    ADDRESS="core-live-a.stellar.org:11625"
    HISTORY="curl -sf http://history.stellar.org/prd/core-live/core_live_001/{0} -o {1}"
    [[VALIDATORS]]
    NAME="SDF 3"
    HOME_DOMAIN="www.stellar.org"
    PUBLIC_KEY=""
    ADDRESS="core-live-c.stellar.org:11625"
    HISTORY="curl -sf http://history.stellar.org/prd/core-live/core_live_003/{0} -o {1}"
    [[VALIDATORS]]
    NAME="Blockdaemon Validator 3"
    HOME_DOMAIN="stellar.blockdaemon.com"
    PUBLIC_KEY=""
    ADDRESS="stellar-full-validator3.bdnodes.net:11625"
    HISTORY="curl -sf https://stellar-full-history3.bdnodes.net/{0} -o {1}"
    [[VALIDATORS]]
    NAME="Blockdaemon Validator 2"
    HOME_DOMAIN="stellar.blockdaemon.com"
    PUBLIC_KEY=""
    ADDRESS="stellar-full-validator2.bdnodes.net:11625"
    HISTORY="curl -sf https://stellar-full-history2.bdnodes.net/{0} -o {1}"
    [[VALIDATORS]]
    NAME="Blockdaemon Validator 1"
    HOME_DOMAIN="stellar.blockdaemon.com"
    PUBLIC_KEY=""
    ADDRESS="stellar-full-validator1.bdnodes.net:11625"
    HISTORY="curl -sf https://stellar-full-history1.bdnodes.net/{0} -o {1}"
---
# Source: soroban-rpc/templates/soroban-rpc-ingress.yaml
apiVersion: v1
kind: Service
metadata:
  name: soroban-test-soroban-rpc
  namespace: default
  labels:
    app: soroban-test-soroban-rpc
    chart: soroban-rpc-0.1.1
    release: soroban-test
    heritage: Helm
spec:
  type: ClusterIP
  ports:
    - name: soroban-rpc
      port: 8000
      targetPort: 8000
  selector:
    app: soroban-test-soroban-rpc
---
# Source: soroban-rpc/templates/soroban-rpc-sts.yaml
apiVersion: apps/v1
kind: StatefulSet
metadata:
  name: soroban-test-soroban-rpc
  namespace: default
  labels:
    app: soroban-test-soroban-rpc
    chart: soroban-rpc-0.1.1
    release: soroban-test
    heritage: Helm
spec:
  # Currently Soroban RPC doesn't support HA deployments over a single shared data store
  # but, each replica pod in a statefulset will have it's own data store on it's PVC.
  # This statefulset creates two replicas which will be monitored by the readiness probe. 
  # Having two replicas also ensures zero downtime during upgrades as at least one replica is 
  # gauranteed to be running while the other one is upgraded and catching backup to network 
  # after restart
  replicas: 2
  serviceName: soroban-test-soroban-rpc
  selector:
    matchLabels:
      app: soroban-test-soroban-rpc
  template:
    metadata:
      labels:
        app: soroban-test-soroban-rpc
        release: soroban-test
    spec:
      containers:
      - name: soroban-rpc
        image: "docker.io/stellar/soroban-rpc:0.9.3"
        args: ["--config-path", "/config/soroban-rpc.cfg"]
        imagePullPolicy: Always
        volumeMounts:
        - name: config-volume
          mountPath: /config
          readOnly: true
        ports:
        - containerPort: 8000
          name: soroban-rpc
        - containerPort: 6061
          name: admin-port
        readinessProbe:
          exec:
            command:
              - /bin/sh
              - -c
              - |
                curl --location --request POST 'http://127.0.0.1:8000/' \
                  --header 'Content-Type: application/json' \
                  --data-raw '{
                    "jsonrpc": "2.0",
                    "id": 10235,
                    "method": "getHealth"
                  }' | jq -es 'if (. | length) == 0 then null else .[0] end | .result | .status == "healthy" and (.latestLedger - .oldestLedger >= (.ledgerRetentionWindow - 10))' > /dev/null;
          failureThreshold: 2
          periodSeconds: 10
          successThreshold: 2
          timeoutSeconds: 5
      volumes:
      - name: config-volume
        configMap:
          name: soroban-test-soroban-rpc
---
# Source: soroban-rpc/templates/soroban-rpc-ingress.yaml
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: soroban-test-soroban-rpc
  namespace: default
spec:
  tls:
  - secretName: soroban-test-soroban-rpc-cert
    hosts:
    - "example.com"
  rules:
  - host: "example.com"
    http:
      paths:
      - path: /
        pathType: Prefix
        backend:
          service:
            name: soroban-test-soroban-rpc
            port:
              number: 8000

```

Reply:
Doh! Thanks, nice catch!